configurator/src/lnd.conf.template

```diff
@@ -57,12 +58,12 @@ tor.skip-proxy-for-clearnet-targets={tor_enable_clearnet}
 tor.streamisolation={tor_stream_isolation}
 
 [watchtower]
-watchtower.active=false
+watchtower.active=true
```

Comment:
Shouldn't this be off if the user has watchtowers off?

Reply:
I did not set a on/off toggle for watchtowers. I just set it to true, and if they want to use it they can add a URI.

Reply:
Seems we should allow the user to disable watchtowers

Reply:
added.